tox.ini

```diff
@@ -3,6 +3,14 @@ envlist = qa, docs, py{36,37,38,39}-{nocov,cov,diffcov}
 skip_missing_interpreters = True
 
 [testenv]
+# One virtualenv per Python version
+envdir =
+    py36: {toxworkdir}/3.6
+    py37: {toxworkdir}/3.7
+    py38: {toxworkdir}/3.8
+    py39: {toxworkdir}/3.9
+    pypy3: {toxworkdir}/pypy3
+    py: {toxworkdir}/python3
```

Comment:
Speeds up GA checking since virtualenv is now shared between "qa", "docs", and "py" (the latter means to use whatever Python version GA has provided)